liquibase.yaml

```diff
@@ -60,6 +85,94 @@ pipeline:
+
+      # Check against the standard expected contents
+      echo "Comparing with expected-distribution-contents.txt"
+      # Filter out commercial components from expected files since we build with !liquibase-commercial
+      grep -v 'liquibase-commercial.jar' liquibase-dist/expected-distribution-contents.txt | sort > expected-filtered.txt
+
+      if diff -u expected-filtered.txt actual-contents.txt; then
+        echo "✓ Distribution contents match expected"
+      else
+        echo "✗ Distribution contents differ from expected."
+        echo "Missing files:"
+        comm -13 actual-contents.txt expected-filtered.txt
+        echo "Extra files:"
+        comm -23 actual-contents.txt expected-filtered.txt
+        exit 1
+      fi
+
+      # Save the timestamps from when we created the markers
+      EXPECTED_CORE_MARKER=$(cat liquibase-core/src/main/resources/WOLFI_MARKER.txt)
+      EXPECTED_STANDARD_MARKER=$(cat liquibase-standard/src/main/resources/WOLFI_STANDARD.txt)
+
+      # Extract the tar.gz archive to a temp folder
+      mkdir -p /tmp/liquibase-check
+      tar -xzf liquibase-dist/target/liquibase-${{package.version}}.tar.gz -C /tmp/liquibase-check
+      cd /tmp/liquibase-check
+
+      # Check 1: WOLFI_MARKER in liquibase-core.jar matches exactly
+      echo "==> Checking liquibase-core.jar for markers"
+      jar -xf internal/lib/liquibase-core.jar WOLFI_MARKER.txt 2>/dev/null || true
+      if [ -f WOLFI_MARKER.txt ]; then
+        ACTUAL_CORE_MARKER=$(cat WOLFI_MARKER.txt)
+        echo "Expected core marker: $EXPECTED_CORE_MARKER"
+        echo "Actual core marker:   $ACTUAL_CORE_MARKER"
+        if [ "$ACTUAL_CORE_MARKER" = "$EXPECTED_CORE_MARKER" ]; then
+          echo "✓ WOLFI_MARKER matches exactly - using our build"
+        else
+          echo "✗ WOLFI_MARKER doesn't match - not using our build!"
+          exit 1
+        fi
+      else
+        echo "✗✗✗ FAILURE: WOLFI_MARKER.txt not found in liquibase-core.jar"
+        jar -tf internal/lib/liquibase-core.jar | head -20
+        exit 1
+      fi
+
+      # Check 2: WOLFI_STANDARD marker in liquibase-core.jar (standard classes are merged into core)
+      jar -xf internal/lib/liquibase-core.jar WOLFI_STANDARD.txt 2>/dev/null || true
+      if [ -f WOLFI_STANDARD.txt ]; then
+        ACTUAL_STANDARD_MARKER=$(cat WOLFI_STANDARD.txt)
+        echo "Expected standard marker: $EXPECTED_STANDARD_MARKER"
+        echo "Actual standard marker:   $ACTUAL_STANDARD_MARKER"
+        if [ "$ACTUAL_STANDARD_MARKER" = "$EXPECTED_STANDARD_MARKER" ]; then
+          echo "✓ WOLFI_STANDARD matches exactly - using our build"
+        else
+          echo "✗ WOLFI_STANDARD doesn't match - not using our build!"
+          exit 1
+        fi
+      else
+        echo "✗✗✗ FAILURE: WOLFI_STANDARD.txt not found in liquibase-core.jar"
+        exit 1
+      fi
```

Comment:
These two checks could potentially be condensed into either a for loop or a helper function, to reduce repetition.